csharp/src/Ice/Discovery/Lookup.cs

```diff
@@ -40,7 +39,7 @@ internal class Lookup : IAsyncLookup
                 catch (Exception ex)
                 {
                     current.Communicator.Logger.Warning(
-                        $"{_pluginName} failed to send foundAdapterById to `{reply}':\n{ex}");
+                        $"Ice.Discovery failed to send foundAdapterById to `{reply}':\n{ex}");
```

Comment:
unrelated to this PR, wouldn't be more correct to say 
```
"Ice.Discovery failed to invoke foundAdapterById on `{reply}'"
```

Reply:
send seems more correct to describe a oneway UDP request.